challenges-31-plus.yml

```diff
@@ -0,0 +1,85 @@
+frame_redirect:
```

Comment:
Is this key on purpose? It doesn't really matter - but it doesn't seem to describe the question